app/v1/controllers/requestParsers/validators/validations/AmendConsolidatedExpensesValidation.scala

```diff
@@ -16,17 +16,22 @@
 
 package v1.controllers.requestParsers.validators.validations
 
-//
-//import v1.models.errors.{MtdError, RuleBothExpensesSuppliedError}
-//import v1.models.request.amendPeriodic._
-//
-//object AmendConsolidatedExpensesValidation {
-//
-//  def validate(consolidatedExpenses: Option[ConsolidatedExpenses], expenses: Option[PeriodDisallowableExpenses]): List[MtdError] = {
-//    (consolidatedExpenses, expenses) match {
-//      case (Some(_), Some(_)) => List(RuleBothExpensesSuppliedError)
-//      case _                  => NoValidationErrors
-//    }
-//  }
-//
-//}
+
+import v1.models.errors.{MtdError, RuleBothExpensesSuppliedError}
+import v1.models.request.amendPeriodic._
+
+object AmendConsolidatedExpensesValidation {
```

Comment:
Please add a corresponding `AmendConsolidatedExpensesValidationSpec` (currently commented out). Note that *any* expenses (allowable or disallowable) along with consolidated expenses should not pass validation. (Thus the current implementation will probably need fixing up once you have some tests.)